serving/samples/grpc-ping-go/client/client.go

```diff
@@ -16,14 +16,17 @@ import (
 
 var (
 	serverAddr         = flag.String("server_addr", "127.0.0.1:8080", "The server address in the format of host:port")
-	serverHostOverride = flag.String("server_host_override", "grpc.knative.dev", "")
+	serverHostOverride = flag.String("server_host_override", "", "")
 	insecure           = flag.Bool("insecure", false, "Set to true to skip SSL validation")
 )
 
 func main() {
 	flag.Parse()
 
-	opts := []grpc.DialOption{grpc.WithAuthority(*serverHostOverride)}
+	var opts []grpc.DialOption
+	if *serverHostOverride != "" {
+		opts = append(opts, grpc.WithAuthority(*serverHostOverride))
```

Comment:
I wonder if this should `log.Fatal`?

Reply:
I'm not sure if understand why we'd fail when host is not specified.

Currently it looks like knative doesn't work without host header (https://github.com/knative/serving/issues) but it ideally should. it's kinda ridiculous to specify an extra hostname when I  already have my own domain name working.

Reply:
I see, this is for the case where you are simply using DNS for the other path?

Reply:
Yeah this is when hostname with proper domain name is inferred from `-server_addr` flag.